tests/sentry/monitors/endpoints/test_organization_monitor_details.py

```diff
@@ -26,9 +26,9 @@ def test_monitor_environment(self):
         monitor = self._create_monitor()
         self._create_monitor_environment(monitor)
 
-        self.get_success_response(self.organization.slug, monitor.guid, environment="production")
+        self.get_success_response(self.organization.slug, monitor.slug, environment="production")
```

Comment:
Can't use guid lookup for these endpoints, this was just working because of the fact that slugs and guids used to be the same, but now autogenerated slugs are different from guids.

ofc, existing monitors will still work fine since their slugs match the old guids.